c/src/ml-api-service-remote.c

```diff
@@ -549,17 +553,21 @@ _mlrs_create_edge_handle (_ml_remote_service_s * remote_s,
  * @brief Internal function to release ml-service remote data.
  */
 int
-ml_service_remote_release_internal (void *priv)
+ml_service_remote_release_internal (ml_service_s * mls)
 {
-  _ml_remote_service_s *mlrs = (_ml_remote_service_s *) priv;
+  _ml_remote_service_s *mlrs = (_ml_remote_service_s *) mls->priv;
 
+  /* Supposed internal function call to release handle. */
   if (!mlrs)
-    return ML_ERROR_INVALID_PARAMETER;
+    return ML_ERROR_NONE;
 
-  nns_edge_release_handle (mlrs->edge_h);
+  if (mlrs->edge_h) {
+    nns_edge_release_handle (mlrs->edge_h);
+
+    /* Wait some time until release the edge handle. */
+    g_usleep (1000000);
```

Comment:
It's not from this PR.. but.. can you justify this 1 sec delay? (is it safe? and is it not too long? can't we check the status/safety actively?)

Reply:
@gichan-jang and I will check the thread in nns-edge when calling edge-release function.

Reply:
There is no specific reason for 1 sec. I think resource release should be processed synchronously with lock in edge-related thread to remove sleep time.